charts/crowdsec/Chart.yaml

```diff
@@ -87,9 +87,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.9.12
+version: 0.9.13
```

Comment:
Please don't change the version we will do this before a release so please revert this change

Reply:
Done 👍